device-types/Ivanti/PulseSecure-PSA300.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Ivanti
+model: PulseSecure PSA300
+slug: ivanti-pulse-secure-psa300
+part_number: PSA300
+u_height: 1
+is_full_depth: false
+weight: 1.13
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Pulse Secure Appliance Serie Data Sheet](https://secureaccessworks.com/datasheets/PulseSecure_OneSheet_PSAHardware.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
```

Comment:
This is not a C14 on board. It appears to be a `dc-terminal`. Please adjust accordingly

Reply:
You are right. I fixed it.